lib/zoom/actions/webinar.rb

```diff
@@ -20,9 +20,9 @@ module Webinar
                           authentication_domains registrants_confirmation_email
                         ],
                         {
-                            language_interpretation: [
-                            :enable,
-                            interpreters: %i[email languages]
+                          language_interpretation: [
+                          :enable,
+                          interpreters: %i[email languages]
```

Comment:
```suggestion
                            :enable,
                            interpreters: %i[email languages]
```
I believe these two lines should still be indented but other than that, I'm good to merge after that is changed.